ztag/schema.py

```diff
@@ -1183,6 +1213,12 @@ class LintBool(String):
                     "banner": ztag_mssql,
                 }, category="1433/MSSQL"),
             }),
+            Port(3306): SubRecord({
+                "mysql": ztag_mysql,
```

Comment:
Should this be `"mysql"::SubRecord({"banner":ztag_mysql})"` (and similarly for postgres), or are these different from the others?